ci-operator/step-registry/openshift-qe/orion/openshift-qe-orion-commands.sh

```diff
@@ -36,38 +36,39 @@ else
 fi
 
 pip install .
-export  EXTRA_FLAGS=""
-if [[ -n "$UUID" ]]; then
+export EXTRA_FLAGS=" --lookback ${LOOKBACK}d"
+if [[ ! -z "$UUID" ]]; then
     export EXTRA_FLAGS+=" --uuid ${UUID}"
 fi
 if [ ${HUNTER_ANALYZE} == "true" ]; then
- export EXTRA_FLAGS+=" --hunter-analyze"
+    export EXTRA_FLAGS+=" --hunter-analyze"
 fi
 if [ ${JUNIT} == true ]; then
-  export EXTRA_FLAGS+=" --lookback ${LOOKBACK}d"
-  export EXTRA_FLAGS+=" --output-format junit"
-  export EXTRA_FLAGS+=" --save-output-path=junit.xml"
-  export EXTRA_FLAGS+=" --ack /tmp/${VERSION}_${ACK_FILE}"
```

Comment:
The rehearsals failed because you changed the "latest ack" download path here.

Reply:
fixed.